test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
Depends on whether or not you want the test to fail reliably on all (supported) platforms when there's a problem.

If you want it to fail reliably on all the platforms on our current CI when there's a problem, then you need a lot of numbers in that Array because the test is non-deterministic and will pass if the buffer manages to flush before the program exits.

Unfortunately, that may make the test take waaay too long when it succeeds. (I don't actually know how long it takes to run on a Raspberry Pi, for example.)

However, if you're happy just having it fail most of the time--but not all the time!--when a problem is introduced, then yes, two numbers in that Array will probably suffice.

If anyone has ideas for making the test deterministic, that would be great. Not sure it's really possible, short of test doubles and making it a unit test rather than a test-the-actual-system test...